english/dog/meetdug.md

```diff
@@ -0,0 +1,7 @@
+"My name is Dug, I have just met you, and I love you!" the dog began to speak using
+a special collar made by his owner. The dog chased a squirrel so I turned
+back to the man and woman.
+
+[Talk to the woman](../woman/scully.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.